tests/test_import_avro.py

```diff
@@ -78,6 +78,7 @@ def test_import_avro_schema():
         type: string
         required: true
         description: order status
+        name: Status
```

Comment:
`name` is not a valid property in the https://datacontract.com/#field-object Field Object.

Do you mean `title`?

Reply:
Thank you so much for pointing out the Field Object & associated properties of it. I will make that change - I also saw I broke a test, so will review that as well before updating the PR!

Reply:
This part has not been resolved, has it?

Reply:
You're right - I fixed up export but not import.  I made the test fail and then fixed the code. Keen eye. Thank you!